src/main/java/dev/andrylat/vomelianchuk/finances/common/UnsupportedDialog.java

```diff
@@ -0,0 +1,16 @@
+package dev.andrylat.vomelianchuk.finances.common;
+
+public class UnsupportedDialog implements Dialog {
+    static final String INCORRECT_ACTION_NUMBER = "The entered number does not match any action";
+
+    @Override
+    public void start() {
+        System.out.println(INCORRECT_ACTION_NUMBER);
+    }
+
+    @Override
+    public String getDescription() {
+        throw new UnsupportedOperationException();
```

Comment:
I prefer to leave some message with high overview explanation why operation is not supported. And it is best practice for each exception: explain what is going wrong.

Reply:
Fixed here: 65f0d9e